docs/network-flow-visibility.md

```diff
@@ -131,62 +131,64 @@ Please modify them as per your requirements.
-| destinationPodNamespace          | 56506         | 102      | string      |
-| destinationPodName               | 56506         | 103      | string      |
-| sourceNodeName                   | 56506         | 104      | string      |
-| destinationNodeName              | 56506         | 105      | string      |
-| destinationClusterIPv4           | 56506         | 106      | ipv4Address |
-| destinationClusterIPv6           | 56506         | 107      | ipv6Address |
-| destinationServicePort           | 56506         | 108      | unsigned16  |
-| destinationServicePortName       | 56506         | 109      | string      |
-| ingressNetworkPolicyName         | 56506         | 110      | string      |
-| ingressNetworkPolicyNamespace    | 56506         | 111      | string      |
-| ingressNetworkPolicyType         | 56506         | 115      | unsigned8   |
-| ingressNetworkPolicyRuleName     | 56506         | 141      | string      |
-| egressNetworkPolicyName          | 56506         | 112      | string      |
-| egressNetworkPolicyNamespace     | 56506         | 113      | string      |
-| egressNetworkPolicyType          | 56506         | 118      | unsigned8   |
-| egressNetworkPolicyRuleName      | 56506         | 142      | string      |
-| ingressNetworkPolicyRuleAction   | 56506         | 139      | unsigned8   |
-| egressNetworkPolicyRuleAction    | 56506         | 140      | unsigned8   |
-| tcpState                         | 56506         | 136      | string      |
-| flowType                         | 56506         | 137      | unsigned8   |
+| IPFIX Information Element        | Field ID | Type        | Description |
+|----------------------------------|----------|-------------|-------------|
+| sourcePodNamespace               | 100      | string      |             |
+| sourcePodName                    | 101      | string      |             |
+| destinationPodNamespace          | 102      | string      |             |
+| destinationPodName               | 103      | string      |             |
+| sourceNodeName                   | 104      | string      |             |
+| destinationNodeName              | 105      | string      |             |
+| destinationClusterIPv4           | 106      | ipv4Address |             |
+| destinationClusterIPv6           | 107      | ipv6Address |             |
+| destinationServicePort           | 108      | unsigned16  |             |
+| destinationServicePortName       | 109      | string      |             |
+| ingressNetworkPolicyName         | 110      | string      | Name of the ingress network policy applied on the destination Pod for this flow. |
+| ingressNetworkPolicyNamespace    | 111      | string      | Namespace of the ingress network policy applied on the destination Pod for this flow. |
+| ingressNetworkPolicyType         | 115      | unsigned8   | 1 stands for Kubernetes Network Policy. <br> 2 stands for Antrea Network Policy. <br> 3 stands for Antrea Cluster Network Policy. |
```

Comment:
using HTML tags (`<br>`) may screw up website generation. To be safe, please put everything on one line.